### ISSUE TITLE
Pass selected field to Cloud Function script

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-dashboard#2473]
### New Feature / Enhancement Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Current Limitation

Currently, it cannot be determined which field was been selected when a script has been invoked.

### Feature / Enhancement Description

Pass the selected field as a separate parameter to the Cloud Function payload.

For example the dashboard user should be able to invoke a script on a user in a specific field of the selected object:
- Transaction class: buying user, selling user

Knowing only the "transaction" object is not enough to know on which user the Cloud Function should be executed.

### Example Use Case

```js
Parse.Cloud.define('deleteUser', async (req) => {
  const selectedTransaction = req.params.object;
  selectedTransaction.set('invalid', true);
  await selectedTransaction.set.save(null, { useMasterKey: true });

  const selectedUser = req.params.selectedField;
  if (selectedUser instanceof Parse.User) {
    selectedUser.set('deleted', true);
    await selectedUser.save(null, { useMasterKey: true });
  }
});
```

### Alternatives / Workarounds

None.

**Comment by mtrezza** [comment]
@dblythy What do you think?

**Comment by dblythy** [comment]
Sounds good

**Comment by mtrezza** [comment]
To avoid any confusion for future readers, in your PR you're passing the name of the selected field, instead of the value of the selected field as suggested in the [example use case](https://github.com/parse-community/parse-dashboard/issues/2473). I think that makes more sense, because if there are 2 fields of the same type with the same value it wouldn't be possible to know which field was selected.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.2.0-alpha.23](https://github.com/parse-community/parse-dashboard/releases/tag/5.2.0-alpha.23)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0-beta.1](https://github.com/parse-community/parse-dashboard/releases/tag/5.3.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0-alpha.1](https://github.com/parse-community/parse-dashboard/releases/tag/5.3.0-alpha.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.3.0](https://github.com/parse-community/parse-dashboard/releases/tag/5.3.0)
